pandas/core/generic.py

```diff
@@ -5629,6 +5629,12 @@ def clip(self, lower=None, upper=None, axis=None, inplace=False,
             Original input with those values above/below the
             `upper`/`lower` thresholds set to the threshold values.
 
+        Notes
```

Comment:
I think it should be in `References` and not in `Notes`. It wasn't in the documentation for the sprint, for simplicity, but you can check this document: http://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Yea, I agree.  Done.